Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem "pry"
+gem "pry-byebug"
```

Comment:
I think those gems should be group.

e.g.
```rb
group :development, :test do
...
end
```

or,  we should use `add_development_dependency` in `gemspec`.